lib/events/index.js

```diff
@@ -996,7 +1052,12 @@ export default function (self) {
     ) {
       return;
     }
-    var pos = self.getLayerPos(e);
+    var pos = self.getLayerPos(e),
+      bm = self.style.gridBorderCollapse === 'collapse' ? 1 : 2,
```

Comment:
What is `bm` here? Is there a more descriptive name possible?

Reply:
http://prntscr.com/20sd870
I referred original code.